format

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+#
+# Script uses clang-format to format all of the source files in a
+# default format.
+#
+
+# The absolute path of the directory containing this script.
+DIR="$( cd "$( dirname "$0" )" && pwd)"
+# Define top level project directory
+PROJECT_DIR="${DIR}"
+
+METHOD="GIT"
+if [ -n "$1" ]; then
+ METHOD=$1
+fi
+
+# Paths to format
+MONITOR=()
+MONITOR+=( "${PROJECT_DIR}/Source" )
+
+if [ "$METHOD" == "ALL" ]; then
```

Comment:
Are we planning to run this script for "all" anytime soon? If not then maybe don't include this mode for now for simpler script?

Reply:
I guess, the right question would be: do we want to run this script for "all" anytime soon? Having a huge diff that touches so many files will affect the usability of git blame/history. I think a much more lenient approach would be to have it run on new PRs for a while, and then run "all" months/a year from now. What do you think?

Reply:
Also, can you share a new diff with the latest format? a08cc7b is outdated right?

Reply:
The plan to run it all at once together with the introduction of the script.

Correct it will affect to the blame / history, but having it one at for all I think would make more sense as gradually and having problems with blame / history one file after another.

https://github.com/TextureGroup/Texture/commit/a08cc7b850950d6202a6c2e997086d1a7c51c7f3 is the latest, I updated the diff with the latest changes morning.

Reply:
ok. In that case, what do you think about keeping the open brackets as is?

Reply:
I think `git blame -w` would sidestep most of the pain, but that's only when running it from the command-line, for i.e. github, sourcetree, IntelliJ, where I think most people actually use it, there's no way to inject that flag into the command :(